react-portal-employees/dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html><html><head><meta charset="utf-8"/><title>Webpack App</title><script defer="defer" src="./main.js"></script><script defer="defer" src="main.js"></script></head><body><h1>Hello world!</h1><h2>Tip: Check your console</h2></body><script>if ('serviceWorker' in navigator) {
+      window.addEventListener('load', () => {
+        navigator.serviceWorker
+          .register('service-worker.js')
+          .then((registration) => {
+            console.log('Service Worker registered: ', registration);
+          })
+          .catch((registrationError) => {
+            console.error('Service Worker registration failed: ', registrationError);
```

Comment:
I think that we need to gitignore this folder

Reply:
Done!